src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1245,4 +1249,260 @@ public SBApiResponse workflowBulkUpdateTransitionV1(String userAuthToken, Multip
+			Map<String, Object> mdoDetails = this.getUserSearchDetails(userIdSet,true, null);
+			Map<String, Object> mdoInfo = (Map<String, Object>) mdoDetails.get(mdoUserId);
+			String departmentName = (String) mdoInfo.get(Constants.ROOT_ORG_NAME);
+			String rootOrgId = (String) mdoInfo.get(Constants.ROOT_ORG_ID);
+			Map<String, Object> allPendingRequest = new HashMap<>();
+			List<WfStatusEntity> pendingRequestsEntityList = wfStatusRepo.getListOfDistinctApplicationUsingDept(Constants.PROFILE_SERVICE_NAME, Constants.SEND_FOR_APPROVAL, departmentName);
+			if(CollectionUtils.isEmpty(pendingRequestsEntityList)) {
+				return ResponseEntity.status(HttpStatus.OK).body("There are no approval requests pending for approval with the MDO");
+			}
+			userIdSet.clear();
+			for (WfStatusEntity wfStatusEntity : pendingRequestsEntityList) {
+				userIdSet.add(wfStatusEntity.getUserId());
+				ObjectMapper objectMapper = new ObjectMapper();
+				List<Map<String, Object>> valuesToBeUpdate = objectMapper.readValue(wfStatusEntity.getUpdateFieldValues(), new TypeReference<List<Map<String, Object>>>() {
+				});
+				List<Map<String, Object>> pendingRequestList;
+				for(Map<String, Object> valueToUpdate : valuesToBeUpdate){
+					if(valueToUpdate.containsKey(Constants.TO_VALUE)){
+						Map<String, Object> updateKeyValue = (Map<String, Object>) valueToUpdate.get(Constants.TO_VALUE);
+						String keyToUpdate = updateKeyValue.keySet().stream().findFirst().get();
+						if(Constants.DESIGNATION.equalsIgnoreCase(keyToUpdate) || Constants.GROUP.equalsIgnoreCase(keyToUpdate)){
+							if(allPendingRequest.containsKey(wfStatusEntity.getUserId())) {
+								pendingRequestList = (List<Map<String, Object>>) allPendingRequest.get(wfStatusEntity.getUserId());
+								pendingRequestList.add(updateKeyValue);
+								allPendingRequest.put(wfStatusEntity.getUserId(), pendingRequestList);
+							} else {
+								pendingRequestList = new ArrayList<>();
+								pendingRequestList.add(updateKeyValue);
+								allPendingRequest.put(wfStatusEntity.getUserId(), pendingRequestList);
+							}
+						}
+					}
+				}
+			}
+			Map<String, Object> allUserDetails = this.getUserSearchDetails(userIdSet,false, rootOrgId);
+			if(MapUtils.isNotEmpty(allUserDetails)){
+				this.populateSheetWithPendingRequests(allPendingRequest, allUserDetails ,pendingRequestWorkBook);
+			} else {
+				return ResponseEntity.status(HttpStatus.OK).body("There are no requests pending for Group or Designation approval with the MDO");
+			}
+			String csvFilePath = "/tmp/pendingRequest.csv";
+			this.convertXSSFWorkbookToCSV(pendingRequestWorkBook, csvFilePath);
+			responseEntity = this.preparePendingRequestFileResponse(csvFilePath);
+		} catch (Exception e) {
+			log.error("An error occurred while downloading file with pending requests", e);
+			responseEntity =  ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+		}
+		return responseEntity;
+	}
+
+
+	private Workbook getBlankWorkseet() {
```

Comment:
We need to return csv file -- not xlsx

Reply:
Xlsx is removed only csv is generated and sent back as reposne.